osu.Game/Database/ImportParameters.cs

```diff
@@ -17,6 +17,11 @@ public struct ImportParameters
         /// </remarks>
         public bool Batch { get; set; }
 
+        /// <summary>
+        /// Whether this import should use copy on write cloning if available.
+        /// </summary>
+        public bool PreferCopyOnWrite { get; set; }
```

Comment:
These parameters are effectively taking the role of "is available" instead of "preferred" now. I was thinking about making the UI text precise about the copy strategy used, thus passing the exact detection result from UI to method choosing.